tests/unit/CorsServiceTest.php

```diff
@@ -164,12 +177,36 @@ public function testHandlePreflightRequest()
         });
 
         $this->service = new CorsService([
+            'allow_origins' => ['http://foo.com'],
+            'allow_methods' => ['post'],
+            'allow_headers' => ['accept', 'authorization', 'content-type'],
+        ]);
+
+        $this->request = new Request;
+
+        $this->specify('regression test for issue #31', function () {
+            $this->request->headers->set('Origin', 'http://foo.com');
+            $this->request->headers->set('Access-Control-Request-Method', 'POST');
+            $this->request->headers->set('Access-Control-Request-Headers', 'accept,authorization,content-type');
```

Comment:
You are not testing with spaces here, can you add a space after at least one of the commas?